api/app/controllers/webhook.controller.ts

```diff
@@ -1,34 +1,38 @@
-const db = require("../sequelize/models");
-const {
-  TRANCRIPT_STATUS,
-  TRANSACTION_TYPE,
+import { Request, Response } from "express";
+
+import { db } from "../sequelize/models";
+import { Review } from "../sequelize/models/review";
+import { Transaction } from "../sequelize/models/transaction";
+import { Transcript } from "../sequelize/models/transcript";
+import { User } from "../sequelize/models/user";
+import { Wallet } from "../sequelize/models/wallet";
+import {
   PR_EVENT_ACTIONS,
+  TRANCRIPT_STATUS,
   TRANSACTION_STATUS,
-} = require("../utils/constants");
-const {
+  TRANSACTION_TYPE,
+} from "../utils/constants";
+import {
   calculateCreditAmount,
   generateTransactionId,
-} = require("../utils/transaction");
-
-const Review = db.review;
-const Transcript = db.transcript;
-const Transaction = db.transaction;
-const User = db.user;
-const Wallet = db.wallet;
+} from "../utils/transaction";
 
 // create a new credit transaction when a review is merged
-async function createCreditTransaction(review, amount) {
+async function createCreditTransaction(
+  review: Review,
+  amount: string | number
+) {
   const dbTransaction = await db.sequelize.transaction();
   const currentTime = new Date();
   const user = await User.findByPk(review.userId);
   const userWallet = await Wallet.findOne({
-    where: { userId: user.id },
+    where: { userId: user?.id },
   });
-  const newWalletBalance = userWallet.balance + +amount;
+  const newWalletBalance = (userWallet?.balance ?? 0) + +amount;
```

Comment:
We don't want to do this assertion `userWallet?.balance ?? 0`. If we can't fetch the balance, we should throw an error